features/ei-analytics-feature/org.wso2.analytics.solutions.ei.analytics.feature/src/main/resources/dashboards/ei-anlaytics-dashboard.json

```diff
@@ -2363,7 +2363,1177 @@
         "hasViewerPermission": false
```

Comment:
Shall we modify the JSON name to `eianalytics.json`?

Reply:
Modified at a3796ee2ff05169450bd135a2b5f3b9d6adef228